src/main/org/deidentifier/arx/AttributeType.java

```diff
@@ -890,6 +891,27 @@ public static MaskingFunction createRandomAlphanumericString(boolean ignoreMissi
                 return new MaskingFunction(new DataMaskingFunctionRandomAlphanumericString(ignoreMissingData, length),
                                            DataScale.NOMINAL, "Random alphanumeric string");
             }
+            
+        /**
+         * Creates a masking function creating a random permutation on column's rows using Fisher-Yates Knuth-Yao (FYKY) algorithm
+         *
+         * @return
+         */
+        public static MaskingFunction createPermutationFunctionColumns() {
+        	return createPermutationFunctionColumns(true,"FYKY");
+        }
+            /**
+             * Creates a masking function creating a random permutation on column's rows
+             * 
+             * @param ignoreMissingData Should the function ignore missing data. Default is true.
+             * @param typePermutation Type of permutation to apply on column's rows
+             * @return
+             */
+            public static MaskingFunction createPermutationFunctionColumns(boolean ignoreMissingData, String typePermutation) {
+        	    return new MaskingFunction(new PermutationFunctionColumns(ignoreMissingData, typePermutation),
+        			                       DataScale.NOMINAL, "Permutation Columns");
+            }
+        

```

Comment:
You also need to add your function to the method AttributeType.listMaskingFunctions()